geographic-crosswalks/geographic-harmonization-guide.qmd

```diff
@@ -0,0 +1,90 @@
+
+-   What are all of the geographic crosswalks we need?
+-   What time perspective should we use? For example, should we report Connecticut data for the nine planning regions before 2022 or use the eight counties?
+-   We use a population threshold to identify large cities. Should this change over time as cities grow and shrink?
+
+## How Geographies Relate from Year-to-Year
+
+Each year some new entities come into existence and other entities are dissolved. This may be the result of new incorporations or statistical delineations, the redrawing or splitting of administrative areas, disincorporations, mergers and consolidations, and other types of changes.
+
+Check out the Census's comphrensive [guide](https://www.census.gov/programs-surveys/geography/technical-documentation/county-changes.2010.html#list-tab-957819518) to all the changes to US census geographies to learn more.
+
+### How many county and county-equivalents are in our data?
+
+We have 3,142 counties and county-equivalents for 2014-2018, 3,143 for 2020-2021, and 3,144 in 2022. The average number of counties per state is 62, but the number varies from three in Delaware to 254 in Texas.
+
+
+```{r}
+#| label: load-output-data
+
+county_data <- read_csv(here::here("data/11_mobility-metrics_county_longitudinal.csv"))
+
+county_data |>
+  count(year)
+```
+
+There are two important changes to note:
+
+#### Alaska county splits
+
+In 2019, the Alaskan county of Valdez-Cordova was split into **Cugach Census Area** and **Copper River Census Area.** Therefore, the number of counties and county-equivalents is 3,143 for 2020 and 2021.
+
+#### Connecticut Planning Regions
+
+The number of US counties increased in 2022 **due to a new county-equivalent map for Connecticut, which better reflected the state's governance system**. The new map divided Connecticut into nine counties, instead of the previous eight. 
+
+Connecticut’s historical eight counties stopped serving as functioning governmental or administrative units in 1960. Currently, Connecticut has nine [Regional Councils of Government](https://portal.ct.gov/OPM/IGPP/ORG/Planning-Regions/Planning-Regions---Overview) (COG) that carry out regional planning and service delivery activities similar to those performed by county-level governments in other states.Planning regions are administrative entities that have have the same boundaries as the state’s COGs.
+
+The Census Bureau used to release data based on Connecticut’s eight counties, but starting in 2023, it switched to reporting data for the nine planning regions, following a 2019 request from the state’s Office of Planning and Management. Since new data products will include planning regions, *we should make sure to use them instead of the historical counties moving forward*. 
+
+See the map below for the planning region boundaries, the towns within each planning region, and historical county boundaries (delineated by thick white borders)
+
```

Comment:
Based on the comparison between prior counties and these planning regions, do you suggest we make a clean break and start just showing the planning regions OR should we try to crosswalk backwards to the counties so we can have a coherent time series for these?

Reply:
I'm going back and forth a bit on whether it would be best to crosswalk past years of data into planning regions, or make a clean break. I'm slightly leaning towards the first option since counties haven't been functional governmental units in CT since 1960. Do you know if there are any data quality concerns related to revising past data to make a consistent time-series? Would this cause problems for users of our data who want to merge it with their own data, or do you think adding an explanation about this change for users would suffice?